eCourse_backend/templates/forms/bs4_text_form.html

```diff
@@ -0,0 +1,63 @@
+{% comment %} 
+  A template to show Forms that process text-input
+  Uses app widget_tweaks to improve the design of the forms
+
+  !!! shall be surrounded by <form ...> </form> !!!
+  !!! doesn't provide the button !!!
+{% endcomment %}
+
+{% load widget_tweaks%}
+
+{% comment %} show hidden fields 
+    TODO: improve
+{% endcomment %}
+{% for hidden_field in form.hidden_fields %}
+  {{hidden_field}}
+{% endfor %}
```

Comment:
no idea how and when hidden fields should be used -> use implementation from template